src/md/tekwill/jf4/homework/Exercise4.java

```diff
@@ -15,8 +19,16 @@
      * 19
      * 13
      */
+    public static void main(String[] args) throws ScriptException
+    {
+        if (args.length == 0)
+        {
+            System.out.println("Looks like you forgot to pass an argument");
+            System.exit(0);
+        }
 
-    public static void main(String[] args) {
-
+        ScriptEngineManager mgr = new ScriptEngineManager();
+        ScriptEngine engine = mgr.getEngineByName("JavaScript");
+        System.out.println(engine.eval(args[0]));
```

Comment:
We do Java, your solution is easy breakable ))

Reply:
At least it gets the math job done and is significantly less laborious compared to other solutions ))